.gitignore

```diff
@@ -47,3 +47,6 @@ config/scout_apm.yml
 config/*.sphinx.conf
 config/unicorn.conf.rb
 lib/tasks/deploy.rake
+
+# macOS resources
+.DS_Store
```

Comment:
There's a note at the top of `.gitignore` about how to hide these files specific to your machine.